platform/security/sts/security-sts-propertyclaimshandler/src/test/java/org/codice/ddf/security/sts/claims/property/TestAttributeFileClaimsHandler.java

```diff
@@ -44,120 +44,315 @@
 
```

Comment:
❗️ Can we rename the class to `AttributeFileClaimsHandlerTest` to match our naming convention for unit test files?